src/components/ComposerQuill/composer.js

```diff
@@ -111,15 +127,24 @@ class Composer extends React.Component {
     }
   }
 
-  handleTextChange() {
-    const {draft, notifyKeyDown} = this.props;
+  handleTextChange(delta, oldDelta, source) {
+    const {notifyKeyDown} = this.props;
+
+    // only do these stuff if user initiated
+    if (source === 'user') {
+      if (notifyKeyDown) {
+        notifyKeyDown();
+      }
 
-    if (notifyKeyDown) {
-      notifyKeyDown();
+      this.saveToDraft();
     }
+  }
+
+  saveToDraft() {
+    const {draft} = this.props;
 
     if (draft?.save) {
-      draft.save(this.quill.root.textContent, draft.id);
+      draft.save(this.quill.getText().trim(), draft.id);
```

Comment:
does .getText() ever return falsy?

Reply:
it returns the string in the composer, which could be `''`